test/ng/directive/ngOptionsSpec.js

```diff
@@ -901,8 +949,7 @@ describe('ngOptions', function() {
 
       // Update the properties on the object in the selected array, rather than replacing the whole object
```

Comment:
@lgalfaso - looking at this a little more I am now thinking that in the specific situation of **multiple** + **track by** we should handle the value of the **track by** changing within each of the items in the collection of selected values.